monix-reactive/shared/src/main/scala/monix/reactive/Consumer.scala

```diff
@@ -125,6 +128,18 @@ trait Consumer[-In, +R] extends ((Observable[In]) => Task[R])
     new MapAsyncConsumer[In,R,R2](self, f)
 }
 
+trait PartitionedConsumer[A] {
```

Comment:
I agree on the matter that you won't use `PartitionedConsumer.ignore` much. I also agree that the interface is to top-level. Though by seeing a `Function2[Int, Seq[A], Task[Unit]]` I don't know what the `Int` is, you have to consult the docs for that matter. Maybe add the trait at the Consumer object? In that case it's not at the top level. For now I'll stick with `Function2`